.github/config/kind-matrix-config.json

```diff
@@ -1,17 +1,5 @@
 {
   "configuration": [
-    {
```

Comment:
suggest: Can this be a separate commit?

Reply:
I need to drop this as it would not pass testing - I implemented `crdsv1` api version to get CRD from apiserver, in 1.14/1.15 it was in `v1beta1`  version.